package.json

```diff
@@ -14,8 +14,8 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "fmtCpp": "wpiformat",
-    "fmtJava": "cd choreolib && ./gradlew spotlessApply",
+    "fmtCpp": "wpiformat -f choreolib && wpiformat -f trajoptlib",
+    "fmtJava": "cd choreolib && bash ./gradlew spotlessApply",
```

Comment:
Does this work on Windows?

Reply:
It worked on my windows. I don't know how to make this work cross-platform where node defaults to cmd.exe on windows and a UNIX-style shell on other platforms.